mongodbatlas/data_source_org_id.go

```diff
@@ -42,7 +43,7 @@ func dataSourceMongoDBAtlasOrgIDRead(ctx context.Context, d *schema.ResourceData
 	for idx, role := range apiKeyOrgList.APIKey.Roles {
 		if strings.HasPrefix(role.RoleName, "ORG_") {
 			if err := d.Set("org_id", apiKeyOrgList.APIKey.Roles[idx].OrgID); err != nil {
-				return diag.Errorf(ErrorProjectSetting, `org_id`, root.APIKey.ID, err)
+				return diag.Errorf(projectapikey.ErrorProjectSetting, `org_id`, root.APIKey.ID, err)
```

Comment:
this error seems to be defined in the wrong package

Reply:
yeah this message was not accurate, adjusted and left a new constant to help with consistency in error messages